demo/kubernetes/rook-cluster.yaml

```diff
@@ -4,7 +4,28 @@ metadata:
   name: rook
 spec:
   namespace: rook
-  version: latest
-  dataDirHostPath: 
-  useAllDevices: false
-  deviceFilter:
+  version: master-latest
```

Comment:
`latest` is reserved for stable builds, which we are not ready for still.  See #547 

`master-latest` is what we'd want users of master's docs to be using.  That is the container version that gets published on every commit to master.  So when a developer finishes a PR and gets merged to master, a new container will be automatically built and published to quay.io with the `master-latest` tag.  Because our example yaml is also using this tag, users will get the latest container bits that are in sync with the docs they are following.